Source/WebCore/css/IsImportant.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) 1999-2003 Lars Knoll (knoll@kde.org)
+ * Copyright (C) 2004, 2005, 2006 Apple Inc.
+ * Copyright (C) 2013 Intel Corporation. All rights reserved.
```

Comment:
Too much copypasta?

Reply:
No, we need to retain the original copyright from `CSSProperty.h` because I'm just copying the code.